Adafruit_FONA.cpp

```diff
@@ -97,103 +110,161 @@ boolean Adafruit_FONA::begin(Stream &port) {
     _type = FONA808_V2;
   } else if (prog_char_strstr(replybuffer, (prog_char *)F("SIM808 R13")) != 0) {
     _type = FONA808_V1;
   } else if (prog_char_strstr(replybuffer, (prog_char *)F("SIM800 R13")) != 0) {
     _type = FONA800L;
-  } else if (prog_char_strstr(replybuffer, (prog_char *)F("SIMCOM_SIM5320A")) != 0) {
+  } else if (prog_char_strstr(replybuffer, (prog_char *)F("SIMCOM_SIM5320A")) !=
+             0) {
     _type = FONA3G_A;
-  } else if (prog_char_strstr(replybuffer, (prog_char *)F("SIMCOM_SIM5320E")) != 0) {
+  } else if (prog_char_strstr(replybuffer, (prog_char *)F("SIMCOM_SIM5320E")) !=
+             0) {
     _type = FONA3G_E;
   }
 
   if (_type == FONA800L) {
     // determine if L or H
 
-  DEBUG_PRINT(F("\t---> ")); DEBUG_PRINTLN("AT+GMM");
+    DEBUG_PRINT(F("\t---> "));
+    DEBUG_PRINTLN("AT+GMM");
 
     mySerial->println("AT+GMM");
     readline(500, true);
 
-  DEBUG_PRINT (F("\t<--- ")); DEBUG_PRINTLN(replybuffer);
-
+    DEBUG_PRINT(F("\t<--- "));
+    DEBUG_PRINTLN(replybuffer);
 
     if (prog_char_strstr(replybuffer, (prog_char *)F("SIM800H")) != 0) {
       _type = FONA800H;
     }
   }
 
 #if defined(FONA_PREF_SMS_STORAGE)
-    sendCheckReply(F("AT+CPMS=" FONA_PREF_SMS_STORAGE "," FONA_PREF_SMS_STORAGE "," FONA_PREF_SMS_STORAGE), ok_reply);
+  sendCheckReply(F("AT+CPMS=" FONA_PREF_SMS_STORAGE "," FONA_PREF_SMS_STORAGE
+                   "," FONA_PREF_SMS_STORAGE),
+                 ok_reply);
 #endif
 
   return true;
 }
 
-
 /********* Serial port ********************************************/
-boolean Adafruit_FONA::setBaudrate(uint16_t baud) {
+
+/**
+ * @brief  Set the baud rate that the module will use
+ *
+ * @param baud The new baud rate to set
+ * @return true: success, false: failure
+ */
+bool Adafruit_FONA::setBaudrate(uint16_t baud) {
   return sendCheckReply(F("AT+IPREX="), baud, ok_reply);
 }
 
 /********* Real Time Clock ********************************************/
-
-boolean Adafruit_FONA::readRTC(uint8_t *year, uint8_t *month, uint8_t *date, uint8_t *hr, uint8_t *min, uint8_t *sec) {
+// This RTC setup isn't fully operational
+// see https://forums.adafruit.com/viewtopic.php?f=19&t=58002&p=294235#p294235
+/**
+ * @brief Read the Real Time Clock
+ *
+ * @param year pointer to a uint8_t to be set with year data
+ * @param month pointer to a uint8_t to be set with month data
+ * @param day pointer to a uint8_t to be set with day data
+ * @param hr pointer to a uint8_t to be set with hour data
+ * @param min pointer to a uint8_t to be set with minute data
+ * @param sec pointer to a uint8_t to be set with year data
+ * @return true: success, false: failure
+ */
+bool Adafruit_FONA::readRTC(uint8_t *year, uint8_t *month, uint8_t *day,
+                            uint8_t *hr, uint8_t *min, uint8_t *sec) {
   uint16_t v;
   sendParseReply(F("AT+CCLK?"), F("+CCLK: "), &v, '/', 0);
+  // TODO: https://github.com/adafruit/Adafruit_FONA/issues/111
```

Comment:
@ladyada Pretty sure you're aware but this doesn't work fully

Reply:
then add docyumentation that it is not compelte